bluebrain/repo-patches/packages/meson/package.py

```diff
@@ -0,0 +1,8 @@
+from spack.package import *
+from spack.pkg.builtin.meson import Meson as BuiltinMeson
+
+
+class Meson(BuiltinMeson):
+    __doc__ = BuiltinMeson.__doc__
+
+    patch("pgmath.patch", when="target=aarch64:")
```

Comment:
```suggestion
    patch("pgmath.patch", when="target=aarch64: %arm")
```
I don't think we need this for GCC based builds?